ui/components/component-library/avatar-base/avatar-base.stories.js

```diff
@@ -121,10 +123,7 @@ export const Children = (args) => (
       backgroundColor={BackgroundColor.infoMuted}
       borderColor={BorderColor.infoMuted}
     >
-      <i
-        className="fa fa-user"
-        style={{ color: 'var(--color-info-default)' }}
-      />
+      <Icon name={ICON_NAMES.USER} color={Color.infoDefault} />
```

Comment:
Can you use `IconColor.iconDefault` here as the enum `IconColor` will specifically serve icon component ~